components/DeleteCandyMachine.tsx

```diff
@@ -12,17 +12,28 @@ const DeleteCandyMachine: FC<{
     const [transaction, setTransaction] = useState('')
     const { removeAccount } = useRemoveCandyMachineAccount([candyMachineAccount])
     const router = useRouter()
+    const { showNotification } = useNotification()
 
     const deleteCM = async () => {
         try {
             setIsDeleting(true)
             setStatus({ error: false, message: '' })
             const result = await removeAccount(candyMachineAccount)
             result && setTransaction(result.txid)
-            setStatus({ error: false, message: `Candy Machine deleted successfully!` })
+            showNotification({
+                open: true,
+                message: `Candy Machine deleted successfully!`,
```

Comment:
Why not using normal quoted strings here instead of string concatenation types?

Reply:
because I forgot to add the candy machine id :sweat_smile: